File: plugins/geo_fencing/lib/src/database/tracker_dao.dart
import 'package:geo_fencing/src/configs.dart';

import 'dao.dart';

class TrackerDao extends Dao {
  static const String columnId = "id";
  static const String columnLocationId = "location_id";
  static const String columnFrom = "start";
  static const String columnTo = "end";

  TrackerDao({required super.appDatabase});

  @override
  String get table => Configs.trackerTable;
}

File: plugins/geo_fencing/android/src/main/kotlin/com/example/geo_fencing/database/TrackerDao.kt
package com.example.geo_fencing.database

class TrackerDao(appDatabase: AppDatabase) : Dao(appDatabase) {
    companion object {
        const val COLUMN_ID = "id"
        const val COLUMN_LOCATION_ID = "location_id"
        const val COLUMN_FROM = "start"
        const val COLUMN_TO = "end"
    }
}